ow/launch/power_demo.launch

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
```

Comment:
Does this file need to exist? It looks like it's almost the same thing as common.launch. If you need to have a demo, can it be a small launch file that includes common.launch?

Reply:
Sorry, didn't realize I was tracking that one. I just made it so I could run without gazebo without accidentally changing common.launch in melodic-devel